hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/client/bootstrap/HoodieSparkBootstrapSchemaProvider.java

```diff
@@ -60,7 +60,7 @@ protected Schema getBootstrapSourceSchema(HoodieEngineContext context, List<Pair
           } else if (ORC.getFileExtension().equals(extension)) {
             return getBootstrapSourceSchemaOrc(writeConfig, context, filePath);
           } else {
-            throw new HoodieException("Could not determine schema from the data files.");
+            throw new HoodieException("Could not determine schema from the data files. Only two file types, orc and parquet, are supported.");
           }
```

Comment:
`Could not determine schema from the data files, supported file formats: [ORC, PARQUET].`

Reply:
@danny0405 thanks，danny.  i have update this exception message